app/src/main/java/org/mifos/mobilebanking/ui/fragments/SavingAccountsTransactionFragment.java

```diff
@@ -160,7 +160,7 @@ public void showSavingAccountsDetail(SavingsWithAssociations
         this.savingsWithAssociations = savingsWithAssociations;
         transactionsList = savingsWithAssociations.getTransactions();
 
-        if (transactionsList.size() > 0) {
+        if (transactionsList != null && transactionsList.size() > 0) {
```

Comment:
```suggestion
        if (transactionsList != null && !transactionsList.isEmpty()) {
```

Reply:
Okay @luckyman20, I'll take care of it from next time. I've made the requested changes. Please have a look.